src/components/Nav/index.js

```diff
@@ -420,7 +420,7 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
                   isDarkTheme ? "Switch to Light Theme" : "Switch to Dark Theme"
                 }
               >
-                <NavIcon name={isDarkTheme ? "darkTheme" : "lightTheme"} />
+                <NavIcon name={!isDarkTheme ? "darkTheme" : "lightTheme"} />
```

Comment:
Rather than using a logical not here, I think it makes more sense to switch the expressions.

- Instead of saying: if it ISNT dark theme, show dark theme icon, else show light theme icon.
- We should say: if it IS dark theme, show the light theme icon, else show the dark theme icon.

```suggestion
                <NavIcon name={isDarkTheme ? "lightTheme" : "darkTheme"} />
```

Reply:
Either way, works well!